### PR TITLE
Prune touchforms api

### PR DESCRIPTION
**Pull request by dannyroberts**
got rid of two more unused cloudcare endpoints

### CODE REVIEW

File: corehq/apps/cloudcare/views.py [hunk 1]
```diff
@@ -577,37 +557,6 @@ def post(self, request, domain):
         })
 
 
-@cloudcare_api
-def render_form(request, domain):
```
Comment (benrudolph) [inline corehq/apps/cloudcare/views.py]: we still use this from formplayer

**Review by benrudolph — COMMENTED** [review]
one thing that needs to be added back but the others i think are good to go

File: corehq/apps/cloudcare/views.py (hunk 1 above)
Comment (benrudolph) [inline corehq/apps/cloudcare/views.py]: https://github.com/dimagi/formplayer/blob/master/src/main/java/services/impl/FormattedQuestionsServiceImpl.java#L32

Comment (benrudolph) [inline corehq/apps/cloudcare/views.py]: (sorry maybe a red herring)

Comment (dannyroberts) [inline corehq/apps/cloudcare/views.py]: we offlined, here's the convo:

> [1:32 PM] 
> Danny Roberts `grep -R 'cloudcare/api/render_form/' .` is showing up, like, a couple of requests from a year ago
> 
> 
> [1:32 PM] 
> Ben Rudolph hmm
> 
> 
> [1:32 PM] 
> Danny Roberts and `ag cloudcare_render_form` on the codebase shows up nothing
> 
> 
> [1:32 PM] 
> Ben Rudolph ah
> 
> 
> [1:33 PM] 
> i thought `/cloudcare/api/readable_questions`
> 
> 
> [1:33 PM] 
> was the same thing
> 
> 
> [1:33 PM] 
> guess it’s not

**Pull request merged**